### DISCUSSION TITLE
Add custom actions on button page

### DISCUSSION THREAD

**Discussion by mcrodriguezb** [Blackymas/NSPanel_HA_Blueprint#2173] [Q&A] [answered]
Hi,

I would like to know if it is possible to define buttons as custom actions. I want to have a button to move up my blinds and another to move down. 

I've seen custom actions are supported on hardware buttons, but are they supported on button pages?

Thanks,

**Comment by edwardtfn** [comment, marked as answer]
Custom actions are not yet supported on button pages, however, you can put your actions in a script and assign that script to any button. 😉
